packages/mobile/locales/es-419/walletFlow5.json

```diff
@@ -38,7 +38,7 @@
   "welcomeToCeloPayments": "Bienvenido a Pagos Celo",
   "letsGetStartedByFirstTransaction": "Empezemos enviando tu primera transacción",
   "sendMoney": "Enviar Dinero",
-  "verificationFee": "Tasa de Verificación",
+  "verificationFee": "Tasa de Verificación",
```

Comment:
i wonder what is different?

Reply:
I resolved the conflict within the GitHub UI, maybe it changed the underlying unicode char 🤷‍♂️